justfile

```diff
@@ -231,3 +231,15 @@ test-go:
     #!/usr/bin/env bash
     export LD_LIBRARY_PATH=$PWD/sdks/go/verification/target/lib:$LD_LIBRARY_PATH
     cd sdks/go && go test -v ./...
+
+contracts-test-echidna *args:
+    nix develop .#echidna -c echidna contracts/test/StakeTableV2.echidna.sol --contract StakeTableV2EchidnaTest --config contracts/echidna.yaml {{args}}
+
+contracts-test-forge *args='-vv':
+    forge test --no-match-test "testFuzz_|invariant_" {{args}}
+
+contracts-test-fuzz *args='-vv':
+    forge test --match-test testFuzz {{args}}
+
+contracts-test-invariant *args='-vv':
+    forge test --match-test invariant_ {{args}}
```

Comment:
do we want to continue with the same naming convention or no need? e.g. testInvariant

Reply:
This `invariant_` prefix is from forge: https://getfoundry.sh/forge/advanced-testing/invariant-testing/